mod-audit-server/src/main/java/org/folio/dao/inventory/impl/InventoryEventDaoImpl.java

```diff
@@ -0,0 +1,69 @@
+package org.folio.dao.inventory.impl;
+
+import static java.lang.String.format;
+import static org.folio.util.DbUtils.formatDBTableName;
+
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.json.JsonObject;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import io.vertx.sqlclient.Tuple;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.dao.inventory.InventoryAuditEntity;
+import org.folio.dao.inventory.InventoryEventDao;
+import org.folio.util.PostgresClientFactory;
+
+public abstract class InventoryEventDaoImpl implements InventoryEventDao {
```

Comment:
It's strange to see abstract and "Impl" in one line :)

Reply:
Well, it contains implementation of interface methods :)